packages/ava/tsconfig.json

```diff
@@ -5,5 +5,5 @@
     "declaration": true,
     "declarationDir": "./lib"
   },
-  "include": ["./src"]
+  "include": ["./src", "__tests__/data"]
```

Comment:
```suggestion
  "include": ["__tests__"]
```

Reply:
fixed. 🤔 I have no impression. It may be an auto change.